#! /bin/sh /usr/share/dpatch/dpatch-run
## link_system_flickrnet.dpatch by Tim Retout <tim@retout.co.uk>
## ported to new upstream by Chris Coulson <chrisccoulson@googlemail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Link against system flickrnet-2.1.5 rather than bundled f-spot version.

@DPATCH@
--- a/configure.ac
+++ b/configure.ac
@@ -323,7 +323,6 @@
 extensions/Exporters/CDExport/Makefile
 extensions/Exporters/DefaultExporters/Makefile
 extensions/Exporters/FlickrExport/Makefile
-extensions/Exporters/FlickrExport/FlickrNet/Makefile
 extensions/Exporters/GalleryExport/Makefile
 extensions/Exporters/FacebookExport/Makefile
 extensions/Exporters/FolderExport/Makefile
--- a/extensions/Exporters/FlickrExport/Makefile.am
+++ b/extensions/Exporters/FlickrExport/Makefile.am
@@ -19,16 +19,13 @@
 	$(LINK_GPHOTO2)				\
 	$(LINK_GIOSHARP)				\
 	$(LINK_GTKSHARPBEANS)		\
-	-r:FlickrNet/FlickrNet.dll	\
 	-r:Mono.Posix
 
 PKGS =					\
+	-pkg:flickrnet-2.1.5		\
 	-pkg:gtk-sharp-2.0		\
 	-pkg:glade-sharp-2.0
 
-SUBDIRS = 			\
-	FlickrNet
-
 RESOURCES =				\
 	-resource:$(srcdir)/$(PLUGIN_MANIFEST)	\
 	-resource:$(srcdir)/$(PLUGIN_NAME).glade
@@ -39,7 +36,6 @@
 	mautil p $(PLUGIN_ASSEMBLY)
 
 $(PLUGIN_ASSEMBLY): $(PLUGIN_SOURCES) $(PLUGIN_MANIFEST)
-	$(MAKE) -C $(SUBDIRS)
 	$(CSC_LIB) -out:$@ $(PLUGIN_SOURCES) $(REFS) $(PKGS) $(ASSEMBLIES) $(RESOURCES)
 
 plugindir = $(pkglibdir)/extensions
